game-core/src/main/java/games/strategy/engine/data/export/GameDataExporter.java

```diff
@@ -119,6 +119,7 @@ private static String playertechs(final GameData data) {
             for (final String definedName : TechAdvance.ALL_PREDEFINED_TECHNOLOGY_NAMES) {
               if (definedName.equals(name)) {
                 name = cat;
+                break;
```

Comment:
Any thoughts to using java stream API here?

Reply:
Found something even simpler